SettingsList.py

```diff
@@ -4041,6 +4054,83 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             "hide_when_disabled": True,
         },
     ),
+    Combobox(
+        name           = 'empty_dungeons_mode',
+        gui_text       = 'Empty dungeons',
+        default        = 'none',
+        choices        = {
+            'none':       'Off',
+            'specific':   'Specific Dungeons',
+            'count':      'Count',
+        },
+        gui_tooltip    = '''\
+            Empty dungeons are dungeons guaranted to be barren, and whose rewards
```

Comment:
For the sake of clarity, I think it'd be better to explicitly call them "Dungeon Rewards" instead of "rewards".

"guaranteed" is with two e's